docs/aggsender.md

```diff
@@ -162,5 +219,7 @@ This paragraph explains different use cases with outcomes.
 
 ## Additional Documentation
 
-[1] https://potential-couscous-4gw6qyo.pages.github.io/protocol/workflow_centralized.html 
-[2] https://agglayer.github.io/agglayer/pessimistic_proof/index.html	
+[1] AggLayer integration guide_WIP
```

Comment:
Is there any URL that hosts this guide, so that we can provide it here?

Reply:
No, I will remove this line for now.